packages/blade/src/components/Accordion/__tests__/Accordion.ssr.test.tsx

```diff
@@ -1,10 +1,34 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
-import { AccordionItem } from '../AccordionItem';
-import { Accordion } from '../Accordion';
+import { Accordion, AccordionItem, AccordionItemBody, AccordionItemHeader } from '../index';
 import renderWithSSR from '~utils/testing/renderWithSSR.web';
 
 describe('<Accordion />', () => {
   it('should render Accordion on server', () => {
+    const { container } = renderWithSSR(
+      <Accordion>
+        <AccordionItem>
+          <AccordionItemHeader title="How can I setup Route?" />
+          <AccordionItemBody>
+            You can use Razorpay Route from the Dashboard or using APIs to transfer money to
+            customers. You may also check our docs for detailed instructions.
+          </AccordionItemBody>
+        </AccordionItem>
+        <AccordionItem>
+          <AccordionItemHeader title="How can I setup QR Codes?" />
+          <AccordionItemBody>
+            Just use Razorpay. You may also check our docs for detailed instructions. Please use the
+            search functionality to ask your queries.
+          </AccordionItemBody>
+        </AccordionItem>
+      </Accordion>,
+    );
+
+    expect(container).toMatchSnapshot();
+  });
+
+  // Skipped because `useId` is rendering different internal ids for `aria-controls`
+  // GitHub issue- https://github.com/razorpay/blade/issues/2072
+  it.skip('should render Deprecated API of Accordion on server', () => {
```

Comment:
render the test twice, this is a known limitation of our ssr test

Reply:
Tried rendering twice but still getting the same error. Also tried rerunning the tests as you mentioned in issue but didnt work